Library/Formula/gnu-complexity.rb

```diff
@@ -0,0 +1,41 @@
+class GnuComplexity < Formula
+  desc "Measures complexity of C source"
+  homepage "https://www.gnu.org/software/complexity"
+  url "http://ftpmirror.gnu.org/complexity/complexity-1.5.tar.xz"
+  mirror "https://ftp.gnu.org/gnu/complexity/complexity-1.5.tar.xz"
+  sha256 "1f5194c0dc6e813ea1dc7f36bfd05f15786a8ad6e9b3ab65b53d2263a0d93102"
+
+  depends_on "autogen"
```

Comment:
No worries. It's a style nit on our part really; these are almost always `:build`-only dependencies so we label them explicitly when they aren't so it doesn't look like they were left runtime accidentally.